apps/biobin/Readme.developer.md

```diff
@@ -1,14 +1,47 @@
 
-Ritchie Lab Software:/BioBin/biobin
-	The DNAnexus-compiled binary of BioBin.
+## Where can I find the BioBin software manual?
+You can find the detailed BioBin software manual here: http://www.ritchielab.com/software/biobin-download
 
-Ritchie Lab Software:/BioBin/biobin-summary.py
-	A helper script which collates and summarizes BioBin output files.
+## What data is required for this app to run?
+This app requires a VCF file to run. If the user has a VCF TBI Index file available, he/she can upload that as an input as well. Otherwise the app will generate the index file for the inputted VCF file. Depending on the type of analysis that the user would like to run (e.g. limiting the analysis to a list of desired genes or to just LOF variants etc.), there are additional input files that need to be included (for more information on which input files to use and on the format of input files, please refer to the software manual above):
+  - Phenotype file (`*.phe`)
+  - Covariate file (`*.cov`)
+  - Role file (`*.rol`)
+  - Weight file (`*.wgt`)
+  - Custom regions file (`*.rgn`) - referred to as "Region file" in the BioBin software manual
+  - Region names list (`*.gen`) - list of gene symbols (1 per line) that the user would like to limit the analysis to. This input file is specific to the app, it is not available in the BioBin command-line application.
+  - BioBin Binary Executable - used only if the user would like to use older versions of the app, should be ignored in most cases.
+  - BioBin Summary Script - used only if the user would like to use older versions of the script that generates the summary output file, should be ignored in most cases.
 
-Ritchie Lab Software:/LOKI/*.db
-	A LOKI knowledge database whose filename indicates the date on which it was built.
+## Usage of the app
+Once you upload all the input files, go to settings where all the available options are listed. Unless you have a reason to change it, keep the "LOKI Knowledge Database" selection as is (the most recent LOKI version is selected by default). If you’d like to do binning followed by an association test, select one of the desired statistical tests: linear regression, logistic regression, wilcoxon test, SKAT-linear (SKAT for continuous phenotypes), or SKAT-logistic (SKAT for categorical phenotypes). In the window with "BioBin arguments" you should include options that you would like to use (specific to the type of analysis that you would like to run). All available options are listed in the manual.
 
+## Examples of analysis type-specific arguments 
+(note - these are just examples; these options are not necessarily complete and should be modified according to user’s preferences):
```

Comment:
Note: these are just examples; these options are not necessarily complete and should be modified according to user’s preferences: